Dockerfile for a tiny container image of AntidoteDB based on Alpine Linux

ilyasToumlilt opened the pull request:
A Dockerfile for a tiny container image (3 times smaller than the default one) of AntidoteDB based on Alpine Linux.

+ This image is used for the Antidote playground
+ Also fixed the Github link in the default Dockerfile

bieniusa commented:
Thanks, this looks neat.
Question: The version on Dockerhub will continue using the old image, right?
Should we change that?

@mweberUKL Any hint/comment from your side?

ilyasToumlilt commented:
Hi,

Yes, the Dockerhub version will continue to be built using the old Dockerfile, were I just updated the link to the git repo.

We should keep both versions on Dockerhub, with different tags. (like they do in https://hub.docker.com/_/erlang/ <https://hub.docker.com/_/erlang/>)
One version from the default setup, and a lightweight one (the Dockefile I added) with -alpine tag.

Ilyas

> On 20 Nov 2018, at 15:19, Annette Bieniusa <notifications@github.com> wrote:
> 
> Thanks, this looks neat.
> Question: The version on Dockerhub will continue using the old image, right?
> Should we change that?
> 
> @mweberUKL <https://github.com/mweberUKL> Any hint/comment from your side?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/AntidoteDB/antidote/pull/358#issuecomment-440289243>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AHU-LGmXlV5KJnj-_3RYQRnatqb-8C9Fks5uxA-OgaJpZM4Yq59t>.
>
